### PR TITLE
fix flutter unit tests (chat input)

### PR DESCRIPTION
**Pull request by gtalha07**

### CODE REVIEW

File: app/lib/features/chat/providers/chat_providers.dart [hunk 1]
```diff
@@ -280,9 +280,9 @@ final allowSendInputProvider = StateProvider.family.autoDispose<bool, String>(
 );
 
 // whether user has enough permissions to send message in room
-final canSendMessageProvider = FutureProvider.family<bool, String>(
+final canSendMessageProvider = FutureProvider.family<bool?, String>(
   (ref, roomId) async {
     final membership = ref.watch(roomMembershipProvider(roomId));
-    return membership.valueOrNull?.canString('CanSendChatMessages') == true;
```
Comment (gtalha07) [inline app/lib/features/chat/providers/chat_providers.dart:286]: This was the one source of tests failing as it seemed our input states logic is dependent on nullable for loading instance. So I had to revert it... (came from Feedback review change)

File: app/lib/features/chat/widgets/custom_input.dart [hunk 2]
```diff
@@ -68,25 +69,29 @@ class CustomChatInput extends ConsumerWidget {
...
-              size: 14,
-              color: Theme.of(context).unselectedWidgetColor,
-            ),
-            const SizedBox(width: 4),
-            Text(
-              key: noAccessKey,
-              L10n.of(context).chatMissingPermissionsToSend,
-              style: TextStyle(
-                color: Theme.of(context).unselectedWidgetColor,
+        child: Padding(
```
Comment (gtalha07) [inline app/lib/features/chat/widgets/custom_input.dart:74]: Also it seemed necessary to move padding inner tree here to avoid overflow...

**Review by bitfriend — APPROVED** [review]
Looks good to me

File: app/lib/features/chat/widgets/custom_input.dart (hunk 2 above)
Comment (gtalha07) [reply app/lib/features/chat/widgets/custom_input.dart:74]: I've seen cases where visually the effects look different comparably. Don't know how it's so but it is what it is...

**Pull request merged**